dissect/target/plugins/os/windows/notifications.py

```diff
@@ -55,19 +192,172 @@ def __init__(self, target):
+            badge_ts=badge_ts,
+            badge_data=badge_data,
+            _target=self.target,
+            _user=user,
+        )
+
+    def _get_appdb_tile_records(self, chunk: c_appdb.Chunk, user: RecordDescriptor) -> list[AppDBTileRecord]:
+        tile_records = []
+        num_tiles = len(chunk.Tiles)
+
+        for tile_no in range(num_tiles):
+            tile = chunk.Tiles[tile_no]
+
+            if tile.UniqueId:
+                tile_arrival_time = None
+                if ts := tile.ArrivalTime:
+                    tile_arrival_time = wintimestamp(ts)
+                tile_expiry_time = None
+                if ts := tile.ExpiryTime:
+                    tile_expiry_time = wintimestamp(ts)
+                name = tile.Name.strip("\x00")
+
+                xml_size = tile.ContentLength
+                tile_xml = chunk.TileXml[tile_no].Content[:xml_size]
+                tile_xml = tile_xml.decode("utf-8", errors="surrogateescape")
+
+                tile_record = AppDBTileRecord(
+                    id=tile.UniqueId,
+                    arrival_time=tile_arrival_time,
+                    expiry_time=tile_expiry_time,
+                    type=tile.Type,
+                    index=tile.Index,
+                    name=name,
+                    content=tile_xml,
+                    _target=self.target,
+                    _user=user,
+                )
+
+                tile_records.append(tile_record)
+        return tile_records
+
+    def _get_appdb_toast_records(self, chunk: c_appdb.Chunk, user: RecordDescriptor) -> list[AppDBToastRecord]:
+        toast_records = []
+        num_toasts = len(chunk.Toasts)
+
+        for toast_no in range(num_toasts):
+            toast = chunk.Toasts[toast_no]
+
+            if toast.UniqueId:
+                toast_arrival_time = None
+                if ts := toast.ArrivalTime:
+                    toast_arrival_time = wintimestamp(ts)
+                toast_expiry_time = None
+                if ts := toast.ExpiryTime:
+                    toast_expiry_time = wintimestamp(ts)
+                name1 = toast.Name1.strip("\x00")
+                name2 = toast.Name2.strip("\x00")
+
+                xml_size = toast.ContentLength
+                toast_xml = chunk.ToastXml[toast_no].Content[:xml_size]
+                toast_xml = toast_xml.decode("utf-8", errors="surrogateescape")
+
+                toast_record = AppDBToastRecord(
+                    id=toast.UniqueId,
+                    arrival_time=toast_arrival_time,
+                    expiry_time=toast_expiry_time,
+                    type=toast.Type,
+                    index=toast.Index,
+                    name1=name1,
+                    name2=name2,
+                    content=toast_xml,
+                    _target=self.target,
+                    _user=user,
+                )
+
+                toast_records.append(toast_record)
+        return toast_records
+
+    @export(record=GroupedRecord)
```

Comment:
```suggestion
    @export(record=[AppDBChunkRecord, AppDBTileRecord, AppDBToastRecord])
```
Might be better, so that we know which records are exported.